compass/Coordinator.java

```diff
@@ -333,37 +341,90 @@ private void createAndBroadcastMilestone(String trunk, String branch) throws Arg
    */
   private boolean validateTransactionsToApprove(String trunk, String branch) {
     if (validatorAPIs.size() > 0) {
-      boolean isConsistent = validatorAPIs.parallelStream().allMatch(api -> {
+
+      return validatorAPIs.parallelStream().allMatch(api1 -> {
```

Comment:
`api1` was named by the IDE - it's a bad name

Reply:
:1st_place_medal: for IntelliJ